build.sbt

```diff
@@ -311,4 +311,4 @@ lazy val docs = project
   .settings(commonSettings)
   .dependsOn(`play-jsonJVM`)
 
-addCommandAlias("validateCode", ";headerCheck;Test/headerCheck;+scalafmtCheckAll;scalafmtSbtCheck")
```

Comment:
`+scalafmtCheckAll` does not make sense (IMHO), `scalafmtCheckAll` does check for all configs anyway. `headerCheck` is the same like `headerCheck;Test/headerCheck`headerCheck;Test/headerCheck